### PR TITLE
[IMP] codeowners: exclude some files for RD-Website

### PR DESCRIPTION
**Pull request by JKE-be**
Using a new team rd-notif-muted

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by rdeodoo — APPROVED** [review]

**Comment by seb-odoo** [comment]
As per code-owners doc, it is not possible to use a negative pattern.
So we introduce a dummy team to serve as catch all to negate some specific patterns.
This is cleaner than trying to adapt the generic website pattern to exclude some modules.
Hopefully some day the dummy team will be replaced by an actual team taking ownership of those.

I just hope github will properly consider the dummy team when parsing the rules even if it has no member. Otherwise we will have to sacrifice someone.

@robodoo r+

File: .github/CODEOWNERS
Comment (seb-odoo) [inline .github/CODEOWNERS:40]: @robodoo r-

Slight adaption though, teams should be prefixed by the organisation, otherwise it is targeting the github user @rd-notif-muted
```suggestion
/addons/website_event*/ @odoo/rd-notif-muted
/addons/website_slides*/ @odoo/rd-notif-muted
```

**Comment by JKE-be** [comment]
well spotted ;)

**Review by seb-odoo — APPROVED** [review]
Should be good now, I guess we'll see after merge if we actually need to create the group or not and/or add members in it.

@robodoo r+

**Comment by JKE-be** [comment]
Team has been already created !

**Pull request closed without merging**
